dotnet-interactive/CommandLine/ReplFrontendEnvironment.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.Interactive.App.CommandLine
+{
+    public class ReplFrontendEnvironment : FrontendEnvironment
```

Comment:
What's this intended to represent? I don't see the `stdio` as being related to a true REPL.

Reply:
to be more explicit this is a Livy scenario.
What the user here is expecting is
 * same rich formatting as browser frontend environment
 * no http api enabled

Reply:
Maybe this is a **RemotedFrontendEnvironment**

Reply:
As far as I can tell this is still a Browser front-end, though with different capabilities.

Reply:
agreed